src/MagnumPlugins/DdsImporter/Test/DdsImporterTest.cpp

```diff
@@ -68,34 +74,41 @@ DdsImporterTest::DdsImporterTest() {
               &DdsImporterTest::dxt3,
               &DdsImporterTest::dxt5,
 
+              &DdsImporterTest::dxt10data,
+              &DdsImporterTest::dxt10tooShort,
+              &DdsImporterTest::dxt10unsupportedFormat,
+
               &DdsImporterTest::useTwice});
+
+    addInstancedTests({&DdsImporterTest::dxt10formats2d}, 46);
+    addInstancedTests({&DdsImporterTest::dxt10formats3d}, 12);
```

Comment:
Even better... I am using `sizeof(that constexpr array in the anonymous namespace)` now ;)